docs/concepts/actions.md

```diff
@@ -77,3 +77,26 @@ Examples:
 
 A great video on the topic is [Good Action Hygiene by Mike Ryan](https://www.youtube.com/watch?v=JmnsEvoy-gY)
 It's for NgRx, but the same naming conventions apply to NGXS.
+
+## Group your actions
+Group similar action by using namespace.
+
+```TS
+export namespace TodoAction {
+  export class Add {
```

Comment:
Please add an empty line between each class and use upper case for the first character in each word of the type.